app/pages/view-solution/controller.js

```diff
@@ -1,13 +1,13 @@
-import { createViewSolutionPageContext } from './context';
+import { createViewSolutionPageContext } from './solutionPageContext';
 import { ApiProvider } from '../../apiProvider';
 import logger from '../../logger';
 
-export const getPublicSolutionById = async (solutionId) => {
-  const response = await new ApiProvider().getPublicSolutionById(solutionId);
+export const getPublicSolutionById = async ({ solutionId }) => {
+  const response = await new ApiProvider().getPublicSolutionById({ solutionId });
 
   if (response.data) {
     logger.info(`Solution ${solutionId}: ${response.data.name} returned`);
-    return createViewSolutionPageContext(response.data);
+    return createViewSolutionPageContext({ previewData: response.data });
```

Comment:
can we call it something else please... `solutionData` perhaps

Reply:
ok